### ISSUE TITLE
Uploading fails on IPv6 hosts

### ISSUE THREAD

**Issue by hobarrera** [asciinema/asciinema#94]
```
~ Asciicast recording finished.
~ Press <Enter> to upload, <Ctrl-C> to cancel.

~ Upload failed, asciicast saved at /tmp/asciicast-447814332
~ Retry later by executing: asciinema upload /tmp/asciicast-447814332
Error: Connection failed (Post https://asciinema.org/api/asciicasts: dial tcp 109.107.38.78:443: network is unreachable)
```

The error message makes me assume that you're assuming that there's native IPv4 connectivity. This assumption is mistaken.

Note: The is a client assumption error, not a network error.

**Comment by sickill** [comment]
This is interesting. Do you still get this error?

There's no explicit use of IPv4 in the code anywhere. I'll check if there is a default in Go's stdlib somewhere for it, maybe that's Go's DNS resolver.

**Comment by hobarrera** [comment]
I could have sworn I'd replied to this pretty fast. My bad.

Any how, I had not noticed this used go at the time of my report. Indeed, go's stdlib isn't really IPv6-ready (hence why I've always said it's not really production-ready for networking apps).

So yeah, asciinema won't work reliably until golang fixes those issues, regrettably.

**Comment by sickill** [comment]
Ok then, I'm leaving it open for now. Thx for the report.

**Comment by sickill** [comment]
@hobarrera I just released v1.3.0 which should solve this issue (as it's implemented in Python). Can you check it out? `sudo pip3 install asciinema`.
